index.js

```diff
@@ -1,4 +1,5 @@
 // The following APIs and libraries are available at runtime:
+// * @google-cloud/bigquery@6.0.1 as google.cloud.bigquery
```

Comment:
Is the leading `@` symbol intentional?

Reply:
Yep! That's the name of the package, for some reason. https://www.npmjs.com/package/@google-cloud/bigquery

Reply:
Huh 🤷

Reply:
https://docs.npmjs.com/about-scopes

It's a scoping thing; npm gives google exclusive access to the `@google-cloud` namespace. Anything not starting with an @ is open season.